### PR TITLE
port to bifrost

### PR DESCRIPTION
**Pull request by cap10morgan**
This eliminates a lot of the boilerplate we've been piling up in these http-api interceptors.

### CODE REVIEW

File: src/voter_registration_http_api/queue.clj [hunk 1]
```diff
@@ -25,35 +24,35 @@
                               ""
                               "voter-registration-works.registration-methods.read"
                               (config [:rabbitmq :queues "voter-registration-works.registration-methods.read"])
-                              response-timeout
+                              5000
```
Comment (tank157) [inline src/voter_registration_http_api/queue.clj]: Are we cool with "magic numbers" now? I mean, it's not that big of a deal, but I'm already seeing different numbers in different projects (election-notification-http-api for instance uses 1000).

Comment (cap10morgan) [inline src/voter_registration_http_api/queue.clj]: No, it should probably be in the config.

**Comment by tank157** [comment]
Yay, code looks good now, I've run the tests, but I'm not sure what to do in krakenstein, so have you tested that out?

**Comment by cap10morgan** [comment]
@tank157 Do you think for this component it's sufficient to curl the endpoints with a voter-registration-works instance in the docker-compose stack? I have done that and they appear to work just fine together.

There's not much else in krakenstein that it would need to interact with, I don't think.

**Comment by tank157** [comment]
Yup, seems fine to me.

**Pull request merged**